openmetadata-ui/src/main/resources/ui/src/components/DataQuality/IncidentManager/TestCaseResultTab/TestCaseResultTab.component.tsx

```diff
@@ -68,7 +68,7 @@ const TestCaseResultTab = () => {
   const { version } = useParams<{ version: string }>();
   const isVersionPage = !isUndefined(version);
   const additionalComponent =
-    testCaseResultTabClassBase.getAdditionalComponents();
+    testCaseResultTabClassBase.getAdditionalComponents(testCaseData);
```

Comment:
We dont need to pass the testCaseData to collate side, it already has access to useTestCaseStore.